source/jormungandr/jormungandr/interfaces/v1/EquipmentReports.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+
+#  Copyright (c) 2001-2017, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+
+from jormungandr import i_manager, timezone
+from jormungandr.interfaces.parsers import default_count_arg_type
+from jormungandr.interfaces.v1.decorators import get_obj_serializer
+from jormungandr.interfaces.v1.errors import ManageError
+from jormungandr.interfaces.v1.ResourceUri import ResourceUri
+from jormungandr.interfaces.v1.serializer import api
+from jormungandr.resources_utils import ResourceUtc
+import six
+
+
+class EquipmentReports(ResourceUri, ResourceUtc):
+    def __init__(self):
+        ResourceUri.__init__(self, output_type_serializer=api.EquipmentReportsSerializer)
+        ResourceUtc.__init__(self)
+        parser_get = self.parsers["get"]
+        parser_get.add_argument("depth", type=int, default=1, help="The depth of your object")
```

Comment:
Don't we want to pass a `filter`argument for ptref as well ?

Reply:
Yes, we want ! 
And it's here :
https://github.com/CanalTP/navitia/pull/2735/files#diff-6a3ab17288956851d997a25b5f61fbcbR81

This is the way for LineReports, for example : 
https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/LineReports.py#L111

Reply:
:ok_man: